manifests/npm.pp

```diff
@@ -8,7 +8,7 @@
   String $package           = $title,
   $source                   = 'registry',
   Array $uninstall_options  = [],
-  $home_dir                 = '/root',
+  $home_dir                 = undef,
```

Comment:
This changes the default behavior of the module and would likely need a major version bump.